doc/start/hardware-recommendations.rst

```diff
@@ -266,34 +296,46 @@ and development clusters can run successfully with modest hardware.
 +--------------+----------------+-----------------------------------------+
 |  Process     | Criteria       | Minimum Recommended                     |
 +==============+================+=========================================+
-| ``ceph-osd`` | Processor      | - 1x 64-bit AMD-64                      |
-|              |                | - 1x 32-bit ARM dual-core or better     |
+| ``ceph-osd`` | Processor      | - 1 core minimum                        |
+|              |                | - 1 core per 200-500 MB/s               |
+|              |                | - 1 core per 1000-3000 IOPS             |
+|              |                |                                         |
+|              |                | * Results are before replication.       |
+|              |                | * Results may vary with different       |
+|              |                |   CPU models and Ceph features.         |
+|              |                |   (erasure coding, compression, etc)    |
+|              |                | * ARM processors specifically may       |
+|              |                |   require additional cores.             |
+|              |                | * Actual performance depends on many    |
+|              |                |   factors including disk, network, and  |
+|              |                |   client throughput and latency.        |
+|              |                |   Benchmarking is highly recommended.   |
 |              +----------------+-----------------------------------------+
-|              | RAM            |  ~1GB for 1TB of storage per daemon     |
+|              | RAM            | - 4GB+ per daemon (more is better)      |
+|              |                | - 2-4GB often functions (may be slow)   |
+|              |                | - Less than 2GB not recommended         |
 |              +----------------+-----------------------------------------+
```

Comment:
Is there an upper limit where adding more RAM does not help anymore?

Reply:
Hi Xahmad,

Really the upper bound is only limited by how many onodes, OMAP, and active data buffers you have.  IE having 2TB of memory for an OSD with a 1TB NVMe drive would be over that upper limit because you'd have more RAM than data over all.  Having 500GB of memory would be more than you'd need to cache onodes (and likely OMAP) but potentially less than you'd need to fully cache a large active data set.  4GB of memory may not be enough to cache active onodes and OMAP depending on various factors (and is much less likely to fully cache the entire active data set).  So generally more memory is better, but the practical benefit typically tapers off fairly quickly after 8-16GB per OSD.